src/SprykerEco/Zed/Adyen/Business/Handler/Notification/AdyenNotificationHandler.php

```diff
@@ -93,6 +93,14 @@ protected function handleNotification(AdyenNotificationRequestItemTransfer $noti
             $this->utilEncodingService->encodeJson($notificationTransfer->getAdditionalData())
         );
 
+        if ($notificationTransfer->getEventCode() === $this->config->getAdyenNotificationEventCodeAuthorisation()) {
+            foreach ($paymentAdyenOrderItems as $item) {
+                if ($item->getStatus() !== $this->config->getOmsStatusNew()) {
```

Comment:
- What about the `authorization failed` state ?, in most state machines from this module we have transitions (`authorized` or `authorization failed`).

Reply:
`authorization failed` is added to available transitions